src/styles/StyleUtils.ts

```diff
@@ -275,28 +281,28 @@ function getZoomSizingStyle(
     containerHeight: number,
     containerWidth: number,
     isLoading: boolean,
-): ViewStyle | CSSProperties | undefined {
+): ViewStyle | undefined {
     // Hide image until finished loading to prevent showing preview with wrong dimensions
     if (isLoading || imgWidth === 0 || imgHeight === 0) {
         return undefined;
     }
-    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px`;
-    const left = `${Math.max((containerWidth - imgWidth) / 2, 0)}px`;
+    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px` as ViewStyle['top'];
+    const left = `${Math.max((containerWidth - imgWidth) / 2, 0)}px` as ViewStyle['left'];
```

Comment:
`import { DimensionValue } from 'react-native'`. It is from **RN**.

Reply:
@rezkiy37 `DimensionValue` doesn't work because it only accept this types: number | "auto" | `${number}%` | Animated.AnimatedNode | null

We are passing `${number}px`

Reply:
However, I can do like this:

```ts
    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px` as DimensionValue;
    const left = `${Math.max((containerWidth - imgWidth) / 2, 0)}px` as DimensionValue;
```

Reply:
Well, makes sense 👍